extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -637,6 +640,25 @@ private static void doServerStart(Vertx vertx, HttpBuildTimeConfig httpBuildTime
         HttpServerOptions sslConfig = createSslOptions(httpBuildTimeConfig, httpConfiguration, launchMode,
                 websocketSubProtocols);
 
+        // Customize
+        if (Arc.container() != null) {
+            InjectableInstance<HttpServerOptionsCustomizer> instances = Arc.container()
+                    .select(HttpServerOptionsCustomizer.class);
```

Comment:
Could you use `listAll()` instead? The `listAll()` contract is guaranted to be ordered by priority.

Reply:
And isn't `select` also enforcing the priority?

The complexity of `listAll` is the same:

```java
List<InstanceHandle<RedisOptionsCustomizer>> customizers = Arc.container().listAll(RedisOptionsCustomizer.class);
for (InstanceHandle<RedisOptionsCustomizer> customizer : customizers) {
    customizer.get().customize(name, options);
}
```

Reply:
Hum, `select` does not mention anything.... so, let's switch (and probably fix reactive messaging too).